services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers_extension.py

```diff
@@ -44,18 +48,18 @@ class DetachContainerFromNetworkItem(_BaseNetworkItem):
 
 @router.patch(
     "/containers/directory-watcher",
-    summary="Enable/disable directory-watcher event propagation",
+    summary="Enable/disable outputs-watcher event propagation",
```

Comment:
minor: I think I would revert this change as well. I am not sure I understand why you changed this everywhere for outputs... since anyway that is the only directory being watched.

Reply:
Making public API display directory-watcher even if internals are slightly different